rustfmt.toml

```
@@ -9,3 +9,4 @@ max_width = 120
 normalize_comments = true
 normalize_doc_attributes = true
 wrap_comments = true
+imports_layout = "Vertical"
```

Missing newline at the end of file:

```suggestion
imports_layout = "Vertical"

```

---

Can we not add this format rule?